### PR TITLE
Fix/712 fix scrolling

### PR DESCRIPTION
**Pull request by JpunktWpunkt**
#712

### CODE REVIEW

File: packages/ui-library/src/components/forms/textarea/index.ts [hunk 1]
```diff
@@ -63,10 +63,28 @@ export class BlrTextarea extends LitElement {
...
       this.count = length;
     }
+
+    const shouldUpdateTextarea = this.textareaElement && scrollTop !== undefined;
+
+    if (shouldUpdateTextarea) {
+      requestAnimationFrame(() => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
+        this.textareaElement.scrollTop = scrollTop;
```
Comment (ChristianHoffmannS2) [inline packages/ui-library/src/components/forms/textarea/index.ts:80]: i quite dont understand why this is needed...

Comment (JpunktWpunkt) [reply packages/ui-library/src/components/forms/textarea/index.ts:80]: "requestAnimationFrame" ensures that the update to the scroll position is synchronized with the browser's rendering cycle. This method waits for the browser to be ready to render the next frame before making changes to the DOM.

"this.textareaElement.scrollTop = scrollTop"; is to preserve the current scroll position of the <textarea>. The scrollTop variable captures the scroll position of the textarea before any update

In summary, this code effectively resolves the Firefox-specific issue of the scrollbar jumping to the top on user input

**Review by bar-tay — APPROVED** [review]

**Review by ChristianHoffmannS2 — APPROVED** [review]
